docs/UserGuide.md

```diff
@@ -113,6 +112,151 @@ Example:
 
 {Example outcome screenshot}
 
+## 4.1 Services Management
+
+This feature allows you to manage the services that your home-based beauty salon provides. You can record the 
+following information about your services: `TITLE`, `DURATION` and `PRICE`.
+
+### 4.1.1 Service Management Command Parameters
```

Comment:
4.1.1 should be on the same level as 4.1.2. Should we reduce the heading level for the commands below from #### to ### to match this?